Gemfile

```diff
@@ -49,6 +47,11 @@ gem "unpwn", "~> 0.3.0"
 # Logging
 gem "lograge"
 
+group :js do
```

Comment:
Why `js` over `assets`? It seems very likely we will be adding css precompilation things here as well. What is the advantage of using separate groups for CSS and js?

Reply:
I think I have mentioned this before. Those are gems having javascript runtime dependency needed only during compilation.

Reply:
Sorry, it was clear what I was trying to clarify. I am trying to understand your reasoning for choosing the group named `js` (over the more usual term `assets`). Would you say calling it `assets` would avoid adding a new group for CSS related gems (ex: sassc-rails)?

Reply:
Moving `sassc-rails` to this group would break compilation of assets in development and production. So this group is really "assets used for production compilation only having JS dependency". Since there's no gem used in dev/test having JS dependency as well, we can do it this way.

I can try to introduce "assets" group instead with all "assets" gems and do following in application.rb. That should work as well.

```ruby
 Bundler.require *Rails.groups(:assets => %w(development test))
```

Reply:
Thanks, I think now I better understand why are you want to call is `js`.
> Moving sassc-rails to this group would break compilation of assets in development 

Correct me if I am wrong, as far as I know `bundler/setup`  will not require a specific group only if `BUNDLE_WITHOUT` is set in `.bundle/config`. We don't mention anything about running `bundle install --without ..` in our development env setup and hence by default all groups are enabled (/required) in the development environment.

Reply:
Yes, but Rails does not use 'bundler/setup'.

https://github.com/rubygems/rubygems.org/blob/2201ad046ce1fe23c5b29b41e56fa41ef1dc96de/config/application.rb#L21

Reply:
oh, I was conflating loaded specs with required files. I am being persistent here because css is working in my development environment even when I move sass-rails to `assets` group, I guess I don't exactly know why. sassc is not in `$LOADED_FEATURES`.